src/containers/Admin/DashboardPickerTab/index.tsx

```diff
@@ -1,15 +1,55 @@
-import React, { useState, useCallback } from 'react'
+import React, { useState, useCallback, useEffect } from 'react'
 import { Heading2 } from '@entur/typography'
 
 import './styles.scss'
+
 import RadioCard from '../../../components/RadioCard'
-import CompactSVG from '../../../assets/previews/Kompakt.svg'
-import ChronoSVG from '../../../assets/previews/Kronologisk.svg'
-import TimelineSVG from '../../../assets/previews/Tidslinje.svg'
+import CompactDark from '../../../assets/previews/previewDark/Kompakt-dark.svg'
+import ChronoDark from '../../../assets/previews/previewDark/Kronologisk-dark.svg'
+import TimelineDark from '../../../assets/previews/previewDark/Tidslinje-dark.svg'
+import CompactLight from '../../../assets/previews/previewLight/Kompakt-light.svg'
+import ChronoLight from '../../../assets/previews/previewLight/Kronologisk-light.svg'
+import TimelineLight from '../../../assets/previews/previewLight/Tidslinje-light.svg'
+import CompactDefault from '../../../assets/previews/previewDefault/Kompakt-blue.svg'
+import ChronoDefault from '../../../assets/previews/previewDefault/Kronologisk-blue.svg'
+import TimelineDefault from '../../../assets/previews/previewDefault/Tidslinje-blue.svg'
+import CompactGrey from '../../../assets/previews/previewGrey/Kompakt-grey.svg'
+import ChronoGrey from '../../../assets/previews/previewGrey/Kronologisk-grey.svg'
+import TimelineGrey from '../../../assets/previews/previewGrey/Tidslinje-grey.svg'
```

Comment:
Her kunne man kanskje gjort slik som i `tavlaLogo.tsx` og hatt en egen komponent for hver visningstype, og sende inn `theme` til hver enkelt komponent så man slipper importere alle de ulike variasjonene her. Koden er lesbar uansett, tenker jeg!